src/sources/statsd/parser.rs

```diff
@@ -18,12 +18,14 @@ static NONALPHANUM: LazyLock<Regex> = LazyLock::new(|| Regex::new(r"[^a-zA-Z_\-0
 #[derive(Clone)]
 pub struct Parser {
     sanitize: bool,
+    convert_timers: bool,
 }
 
 impl Parser {
-    pub const fn new(sanitize_keys: bool) -> Self {
+    pub const fn new(sanitize_keys: bool, convert_timers: bool) -> Self {
```

Comment:
Nit:
This would be more readable if we we have a new `enum` type. Same for all the functions that use this new `bool`. For example:
```suggestion
    pub const fn new(sanitize_keys: bool, convert_to: ConversionUnit) -> Self {
```

Currently the default behavior is to convert to seconds (assuming ms as the incoming value) but we might want to extend it in the future.

Reply:
Okay I will change accordingly but StatsD data usually use only two time units (ms, s) in distribution.

Reply:
@pront Thank you for suggestion! The implementation's done.